examples/juliatypes.jl

```diff
@@ -250,9 +250,10 @@ issub(a::UnionT, b::UnionT, env) = a === b || issub_union(a, b, env, true, env.R
 issub(a::UnionT, b::Ty, env) = b===AnyT || issub_union(b, a, env, false, env.Lunions)
 issub(a::Ty, b::UnionT, env) =
     a===BottomT || a===b.a || a===b.b || issub_union(a, b, env, true, env.Runions)
-# take apart unions before handling vars
-issub(a::UnionT, b::Var, env) = issub_union(b, a, env, false, env.Lunions)
-issub(a::Var, b::UnionT, env) = a===b.a || a===b.b || issub_union(a, b, env, true, env.Runions)
+
+# handle vars before unions
+issub(a::UnionT, b::Var, env) = var_gt(b, a, env)
+issub(a::Var, b::UnionT, env) = var_lt(a, b, env)
```

Comment:
do these changes mean the algorithm in this file fully matches the one in `subtype.c`?